### PR TITLE
Add remove_tag to CLI client

### PR DESCRIPTION
**Pull request by jaegeral**
This pull request adds a new option to the timesketch cli client to remove tags from an event. The new option, called `remove_tag`, takes a list of tags as its argument and removes those tags from the event. For example, the following command would remove the tags "suspicious" and "bad" from the event with the ID "12345".

```
timesketch events remove_tag --timeline-id 4 --event-id k8P1MYcBkeTGnypeeKJL --tag foobar333
200
```

https://github.com/google/timesketch/issues/2605

### CODE REVIEW

File: cli_client/python/timesketch_cli_client/commands/events.py [hunk 1]
```diff
@@ -145,3 +145,57 @@ def add_event(ctx, message, date, attributes, timestamp_desc, output):
...
+
+    if tag:
+        # if tag is a string with commas, make it a list
+        if "," in tag:
+            tags = tag.split(",")
+            return_value = sketch.untag_events([event_id], timeline.index_name, tags)
+        else:
+            return_value = sketch.untag_event(event_id, timeline.index_name, tag)
+            click.echo(return_value)
+    else:  # just get the event
```
Comment (jkppr) [inline cli_client/python/timesketch_cli_client/commands/events.py:193]: Why not check if the event exists first, before going and try to untag something?

**Review by jkppr — APPROVED** [review]
generally looks good. Just one small question below.

File: cli_client/python/timesketch_cli_client/commands/events.py (hunk 1 above)
Comment (jaegeral) [reply cli_client/python/timesketch_cli_client/commands/events.py:193]: Not sure I understand, this is to give the user the ability before removing any tags to call it without the tags argument, you will see the event, and then you can re-run it with tags to be removed.

It is independent if an event exists or not.

Comment (jaegeral) [reply cli_client/python/timesketch_cli_client/commands/events.py:193]: we just checked and we do the same in annotations.

**Pull request merged**